### DISCUSSION TITLE
How to add multiple parameter values for the --kube-apiserver-arg option in the K3s service.

### DISCUSSION THREAD

**Discussion by CrushCCD** [k3s-io/k3s#10723] [Q&A] [answered]
1. How to add multiple parameter values for the `--kube-apiserver-arg` option in the K3s service.
![image](https://github.com/user-attachments/assets/05e82579-9bb1-4c63-854e-d248bb843917)

2. When configuring the `authentication-token-webhook-config-file` parameter value, it can accept the `tokenview` webhook.
webhook-config.json:
![image](https://github.com/user-attachments/assets/dc88cfe1-8631-4fd9-8057-8165de95fa01)

about：
https://www.51cto.com/article/694171.html

The k3s version  :
k3s version v1.25.11+k3s1 (582f07cf)
go version go1.19.10

**Comment by brandond** [comment, marked as answer]
Just use the flag multiple times. If you're using a config file instead of CLI flags, you can pass it a yaml list.
